src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,117 @@
+@import '/src/prepare-styles/colors.scss';
+@import '/src/prepare-styles/mixins.scss'; 
+@import '/src/prepare-styles/usable-fonts.scss'; 
+
+$tablet-min-width: 640px;
+$desktop-min-width: 1200px;
+
+
+
+@mixin content-padding-inline {
+  padding-inline: 16px;
+
+  @include on-tablet {
+    padding-inline: 32px;
+  }
+
+  @include on-desktop {
+    padding-inline: 152px;
+  }
+}
+
+.container {
+  font-family: 'Mont-Regular'; 
+  @include content-padding-inline;
+}
+
+.footer {
+  position: relative;
+  display: block;
+  text-align: left;
+  padding-block: 32px;
+  border-top: 1px solid $grayElements;
+
+  @include on-tablet {
+    flex-wrap: wrap;
```

Comment:
Also, i dont remember can we use flex-wrap without flex(on that case display block)